tests/hooks/benchmark-hooks.sh

```
@@ -0,0 +1,115 @@
+    local hook_path="$2"
+    local test_command="$3"
+    local json_input="{\"tool_name\":\"Bash\",\"tool_input\":{\"command\":\"$test_command\"}}"
+
+    echo "Benchmarking: $hook_name"
+    echo "Test command: $test_command"
+
+    local total_time=0
+    local min_time=999999
+    local max_time=0
+
+    for i in $(seq 1 $ITERATIONS); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)